HLTrigger/special/plugins/HLTMuonRechitClusterFilter.cc

```diff
@@ -0,0 +1,113 @@
+#include "DataFormats/MuonReco/interface/MuonRecHitCluster.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+#include "FWCore/Framework/interface/global/EDFilter.h"
+#include "FWCore/Framework/interface/maker/WorkerMaker.h"
+#include "FWCore/ParameterSet/interface/ConfigurationDescriptions.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/ParameterSet/interface/ParameterSetDescription.h"
+#include "FWCore/ParameterSet/interface/ParameterSetDescriptionFiller.h"
+#include "FWCore/Utilities/interface/EDGetToken.h"
+#include "FWCore/Utilities/interface/InputTag.h"
+#include "FWCore/Utilities/interface/StreamID.h"
+
+class HLTMuonRechitClusterFilter : public edm::global::EDFilter<> {
```

Comment:
```suggestion
class HLTMuonRecHitClusterFilter : public edm::global::EDFilter<> {
```
Since the input objects are called `MuonRecHitCluster` (capital H), please be consistent and use `RecHit` instead of `Rechit` whenever needed (this comment only applies to the code under `HLTrigger/*`).

Reply:
Should I change the file name as well?

Reply:
Yes, please.